spring-boot/backend/src/test/java/example/web/HelloControllerTests.java

```diff
@@ -0,0 +1,13 @@
+package example.web;
+
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class HelloControllerTests {
+
+	@Test
+	void myBadTest() throws Exception {
+		assertThat("asdf").isNotNull();
```

Comment:
Going to keep this test?

Reply:
ah. I'll figure out how to exclude tests in the build. I think its
```
test {
    exclude '**/*'
}
```